test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -291,12 +294,11 @@ class ExtProcIntegrationTest : public HttpIntegrationTest,
     }
   }
 
-  void processAndRespondImmediately(bool first_message,
+  void processAndRespondImmediately(FakeUpstream& grpc_upstream, bool first_message,
```

Comment:
Nit: do you think organizing these gRPC upstream methods inside a class would make sense?

Reply:
Yes, I think that would be cleaner, and allow for more flexible tests in the future with separate processor_connection_ and processor_stream_ for each FakeExternalProcessor.

I'd prefer to do that in a follow-on PR, though, to delineate the test refactor from the behavior additions in this PR.

Reply:
Sure, that's fine.